src/private/WRLDMapView.mm

```diff
@@ -1190,6 +1214,14 @@ - (void)notifyBuildingInformationReceived:(int)buildingHighlightId
     }
 }
 
+- (void)notifyIndoorEntitySelected:(const std::vector<std::string>&)indoorEntityIds
```

Comment:
again, minor naming, elsewhere in ios-api we use "tapped" rather than "selected" I think?
https://github.com/wrld3d/ios-api/blob/master/src/private/WRLDMapView%2BPrivate.h#L23
https://github.com/wrld3d/ios-api/blob/master/src/public/WRLDMapViewDelegate.h#L66